Formula/ciphey.rb

```diff
@@ -0,0 +1,187 @@
+  license "MIT"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "poetry" => :build
+  depends_on "swig" => :build
+  depends_on "libyaml"
+  depends_on "python@3.9"
+  depends_on "six"
+
+  resource "cipheycore" do
+    url "https://github.com/Ciphey/CipheyCore/archive/v0.3.2.tar.gz"
+    sha256 "d05b4c15077b56121e96c1b934ae2d49f14004834a9b8fbc1bdc76782cd66253"
+  end
+
+  resource "appdirs" do
+    url "https://files.pythonhosted.org/packages/d7/d8/05696357e0311f5b5c316d7b95f46c669dd9c15aaeecbb48c7d0aeb88c40/appdirs-1.4.4.tar.gz"
+    sha256 "7d5d0167b2b1ba821647616af46a749d1c653740dd0d2415100fe26e27afdf41"
+  end
+
+  resource "astroid" do
+    url "https://files.pythonhosted.org/packages/6a/7f/90312f42efc2a5249f26e436be10f0c53c512b2f3a36af364cd021283660/astroid-2.6.2.tar.gz"
+    sha256 "38b95085e9d92e2ca06cf8b35c12a74fa81da395a6f9e65803742e6509c05892"
+  end
+
+  resource "base58" do
+    url "https://files.pythonhosted.org/packages/b5/c1/8e77d5389cf1ea2535049e5ffaeb241cce21bcc1c42624b3e8d0fb3bb607/base58-2.1.0.tar.gz"
+    sha256 "171a547b4a3c61e1ae3807224a6f7aec75e364c4395e7562649d7335768001a2"
+  end
+
+  resource "base91" do
+    url "https://files.pythonhosted.org/packages/5b/19/b0f1df635561be1bcd74c652a2e611ebf0a9d5f5f10be9d8421733e2cb04/base91-1.0.1.tar.gz"
+    sha256 "5b284a2ba3c97be1eb9473f3af94a9bf141d61005d836e75e645d2798da58799"
+  end
+
+  resource "cipheydists" do
+    url "https://files.pythonhosted.org/packages/59/bf/0a2069da3364362986095a37929677ec71f8827988408c8bf9d815f32d14/cipheydists-0.3.35.tar.gz"
+    sha256 "3436fde3f57df732e1a65fb03a565a564dd9d0c8d130c2e94f8b852e6a199a88"
+  end
+
+  resource "click" do
+    url "https://files.pythonhosted.org/packages/27/6f/be940c8b1f1d69daceeb0032fee6c34d7bd70e3e649ccac0951500b4720e/click-7.1.2.tar.gz"
+    sha256 "d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a"
+  end
+
+  resource "colorama" do
+    url "https://files.pythonhosted.org/packages/1f/bb/5d3246097ab77fa083a61bd8d3d527b7ae063c7d8e8671b1cf8c4ec10cbe/colorama-0.4.4.tar.gz"
+    sha256 "5941b2b48a20143d2267e95b1c2a7603ce057ee39fd88e7329b0c292aa16869b"
+  end
+
+  resource "commonmark" do
+    url "https://files.pythonhosted.org/packages/60/48/a60f593447e8f0894ebb7f6e6c1f25dafc5e89c5879fdc9360ae93ff83f0/commonmark-0.9.1.tar.gz"
+    sha256 "452f9dc859be7f06631ddcb328b6919c67984aca654e5fefb3914d54691aed60"
+  end
+
+  resource "flake8" do
+    url "https://files.pythonhosted.org/packages/9e/47/15b267dfe7e03dca4c4c06e7eadbd55ef4dfd368b13a0bab36d708b14366/flake8-3.9.2.tar.gz"
+    sha256 "07528381786f2a6237b061f6e96610a4167b226cb926e2aa2b6b1d78057c576b"
+  end
+
+  resource "isort" do
+    url "https://files.pythonhosted.org/packages/b7/8b/7c2200599c22b4ef6f3688f93c4f44065926bc05cbd38c31247b1348f9a3/isort-5.9.1.tar.gz"
+    sha256 "83510593e07e433b77bd5bff0f6f607dbafa06d1a89022616f02d8b699cfcd56"
+  end
```

Comment:
Just FYI, the ciphey team have some dev deps in the runtime deps section, so there's quite a few unnecessary dependencies present in this formula (e.g. flake8, isort, pylint etc). It might be simpler just to allow them in to make the bump action simpler, but that's your call.

_(Just so you're aware, at MacPorts, I only added the necessary deps)_.